index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Microverse Portfolio</title>
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+        />
+        <link rel="stylesheet" href="css/styles.css" />
+    </head>
+    <body>
+        <header class="toolbar">
+            <p class="toolbar__title">JorGSA</p>
+            <nav class="toolbar__menu">
+                <ul class="menu__list">
+                    <li class="menu__item"><a class="menu__link" href="">Portfolio</a></li>
+                    <li class="menu__item"><a class="menu__link" href="">About</a></li>
+                    <li class="menu__item"><a class="menu__link" href="">Contact</a></li>
+                </ul>
+            </nav>
+            <button class="toolbar__btn">
+                <span class="material-symbols-outlined toolbar__icon">menu</span>
+            </button>
+        </header>
+    </body>
+    <section class="headline">
+        <img class="headline__decorator-rectangle" src="img/decorator-rectangle.png" alt="" />
+        <img class="headline__decorator-cube" src="img/decorator-cube.png" alt="" />
+        <img class="headline__decorator-cube-incomplete" src="img/decorator-cube-incomplete.png" alt="" />
+        <img class="headline__decorator-cube-left" src="img/decorator-cube-left.png" alt="" />
+        <img class="headline__decorator-cylinder" src="img/decorator-cylinder.png" alt="" />
+        <img class="headline__decorator-small-cylinder" src="img/decorator-small-cylinder.png" alt="" />
```

Comment:
Kindly, I suggest you put all the decoration images in the background, with background-image, it can be only one and adapt the size to the device or put several backgrounds and position it in their respective locations